labels/labels.go

```diff
@@ -23,3 +23,7 @@ const LabelUncompressed = "containerd.io/uncompressed"
 // LabelSharedNamespace is added to a namespace to allow that namespaces
 // contents to be shared.
 const LabelSharedNamespace = "containerd.io/namespace.shareable"
+
+// LabelDistributionSource is added to content to reference the source
+// repository it was obtained from.
```

Comment:
"source repository" can be misinterpreted as a source code repository on VCS

Reply:
```go
// LabelDistributionSource is added to content to indicate the origin of the blob.
// e.g., "containerd.io/distribution.source.docker.io=library/redis"
```

Reply:
Thanks! Fixed.